tests/platform_tests/test_kdump.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import time
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.processes_utils import wait_critical_processes
+from tests.common.reboot import reboot, SONIC_SSH_PORT, SONIC_SSH_REGEX, wait_for_startup,\
+    REBOOT_TYPE_COLD, REBOOT_TYPE_KERNEL_PANIC
+from tests.platform_tests.test_reboot import check_interfaces_and_services
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('any')
+]
+
+SSH_SHUTDOWN_TIMEOUT = 360
+SSH_STARTUP_TIMEOUT = 420
+
+SSH_STATE_ABSENT = "absent"
+SSH_STATE_STARTED = "started"
+
+
+class TestKernelPanic:
+    """
+    This test case is used to verify that DUT will load kdump crashkernel on kernel panic.
+    """
+    def wait_lc_healthy_if_sup(self, duthost, duthosts, localhost, conn_graph_facts, xcvr_skip_list):
```

Comment:
@amnsinghal is this test supposed to run only on supervisor?

Reply:
No, the test runs on both LCs and RPs. For RPs, there is an additional check to ensure that LCs come up healthy after RP goes down and come back up. (In SONiC distributed chassis, LCs go down as well if RP goes down - Ref: https://github.com/sonic-net/SONiC/pull/945#discussion_r828304573)